src/Network/Network/NetworkWatcher/ConnectionMonitor/MigrateNetworkWatcherClassicConnectionMonitor.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Microsoft.Azure.Commands.Network.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Network;
+using Microsoft.Azure.Management.Network.Models;
+using System;
+using System.Management.Automation;
+using MNM = Microsoft.Azure.Management.Network.Models;
+
+namespace Microsoft.Azure.Commands.Network.NetworkWatcher.ConnectionMonitor
+{
+    [Cmdlet("Migrate", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "NetworkWatcherClassicConnectionMonitor", SupportsShouldProcess = true, DefaultParameterSetName = "SetByName"), OutputType(typeof(PSConnectionMonitorResultV1), typeof(PSConnectionMonitorResultV2))]
+    public class MigrateNetworkWatcherClassicConnectionMonitor : ConnectionMonitorBaseCmdlet
```

Comment:
Please also provide input object parameter set and resourceId parameter set. See the reference https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/design-guidelines/cmdlet-best-practices.md

Reply:
Sure, acknowledging

Reply:
We need this commad in this format, as its consecutive or get command has the same parameters, its easy to understand for the user.
Like 
Get-AzNetworkWatcherConnectionMonitor -NetworkWatcherName "NetworkWatcher_centraluseuap" -ResourceGroupName NetworkWatcherRg -Name "Cmv29Feb1MigrationTaskCM"

Once the user get the CM, user can use the same parameters to migrate the CM to CMv2.

Reply:
@v-sapanchal We do not request you to change the current parameter set but better you can add 2 more parameter sets as mentioned in the above link. You can refer to the cmdlet Get-AzNetworkWatcherConnectionMonitor, which has the parameter sets SetByResource and SetByResourceId. It would be convenient for our users to run it.